### DISCUSSION TITLE
vercel部署失败提示This Serverless Function has crashed

### DISCUSSION THREAD

**Discussion by Nemo137-Liang** [walinejs/waline#1340] [Q&A] [answered]
尝试了两种部署步骤：
1. 使用#1098 discussion里提供的快速部署链接出现该错误；
2. 直接fork example然后在vercel进行import导入，并指定Root Directory为example，依然出现该错误。

![image](https://user-images.githubusercontent.com/60751406/187738519-3807343c-98d3-454e-aa04-46d552c3789a.png)

**Comment by lizheming** [comment, marked as answer]
https://github.com/walinejs/waline/issues/1334
